.github/workflows/create-tag.yml

```diff
@@ -17,23 +17,20 @@ jobs:
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
-      - name: Parse Plugin Version
+      - name: Parse Version
         id: parse-version
         run: |
-          # Parse new tag from AssemblyFileVersion
-          VERSION="$(cat plugin/CactbotEventSource/Properties/AssemblyInfo.cs | sed -ne 's/.*AssemblyFileVersion.*"\([0-9]*\.[0-9]*\.[0-9]*\)\.0".*/\1/p')"
+          # Parse new tag from package.json
+          VERSION="$(cat package.json | sed -ne 's/"version": "\([0-9]*\.[0-9]*\.[0-9]*\)",/\1/p')"
```

Comment:
I think the logic above is not quite correct.  This workflow is looking for versions in the .cs files, but also writes the .cs files??

If you'd like to add the tag to package.json, that seems good.  What about leaving this script mostly as-is and adding an additional check that all three versions (instead of all two) versions match before creating a tag? I think it's an error not to have all of these match.

Reply:
Sorry, maybe I should clarify my purpose first.
It can read version number from `package.json` and then write into .cs files.

When you publish a release, you should always change the version number in two `AssemblyInfo.cs` files, and then let Github Actions run this workflow and release it, right?
In fact, `npm version` is a command that can increase version number properly and do a git commit (also `git-tag` it if you like), so I think it should be easier to run a command like `npm version patch` and push, then Github Actions would help you to change the version numbers in `AssemblyInfo.cs` files and create the corresponding tag.

Reply:
Ah! ⊙﹏⊙∥
I didn't create a git commit that includes .cs files after I changed their version numbers...

Reply:
Yes, but in this patch there would be two commits that say "Bump version number"? One from the `package.json` commit and then one from the workflow, right?

I guess my feeling is that this isn't that hard to do, and that adding a script here makes this more complicated rather than easier?

I am supportive of having a version number in `package.json`.  That sounds great.  If you felt like this needed a script and automation, then I think I would rather have a local script (e.g. `npm run bumpversion 0.23.7`) than a workflow.

Reply:
Done!
There is a hook script in running [`npm version` command](https://docs.npmjs.com/cli/v7/commands/npm-version) called "version", so I can add a js script to sync version number from package.json to AssemblyInfo.cs files.

After that, you can just `npm version patch` or `npm version minor` to increase the version number and automatically commit the bumping change, or something like `npm version 0.22.1` to specify a number, more detail in [npm's doc](https://docs.npmjs.com/cli/v7/commands/npm-version)